libraries/helpers_sensuctl.rb

```diff
@@ -5,6 +5,10 @@ def sensuctl_bin
         '/usr/bin/sensuctl'
       end
 
+      def sensuctl_path
```

Comment:
Why introduce a new method here for windows use? Do you think it will make things easier to maintain?

Reply:
Easier to maintain / use windows wording, but could go either way. I recall the helper not honoring the platform variable, but could have been a poor attempt at implementation and I fell back to separate methods.

Reply:
I think it would be better to use a single function to not have to litter more platform checks in multiple locations. I would ideally opt for something like the old cookbook had: https://github.com/sensu/sensu-chef/blob/develop/libraries/sensu_helpers.rb#L31-L39.